### ISSUE TITLE
PDF Annotation tools on mac OS?

### ISSUE THREAD

**Issue by akirosingh** [BryanJenksCommunity/FAQ#9]
### Question

I've been trying the Preview (Highlight/notes) -> Zotero -> Markdown annotations -> Obsidian templates, but Preview has been very unreliable in saving my highlights and notes. Do you have any alternatives to Preview that is more reliable?

---

By filing a question to this repo, I promise that

-   [x] I have searched open/closed issues on this repo for my answer already

I understand that my issue may be closed if I don't fulfill my promises.

**Comment by tallguyjenks** [comment]
@akirosingh strange, I actually use preview and found that I liked it. Maybe it's a matter of the file copy you're editing is on the local file system and not **in** the Zotero database? when I import files I always add the copy to the Zotero database and never store them out in the file system at large. After annotations are made I always compulsively save the file before the rest of the MD notes workflow. 

doing all of this still yields errors?

on mac, I use primarily preview and haven't had a need to try out anything else yet, sorry!

**Comment by akirosingh** [comment]
@tallguyjenks Yeah, I was working with a Zotero library backed up on Box cloud storage and double-checked the version control, and even though it says there were edits none of the highlights were updated in the saved file. I also manually saved the file to make sure before closing. I redid some of the note-taking and the problem repeated. It could also be PDF specific or the cloud system I used. Looks like it is a known issue: https://discussions.apple.com/thread/8319420

I didn't mean to turn the question into a debugging session! I think I've found skim app as a better pdf annotating tool for me (Faster with lots of notes and highlights, and decent interface) but the plaintext notes and highlight export isn't quick as spiffy as the markdown export through the Zotero plugin. Anyway, I'm enjoying the content you're putting out! Thanks for sharing!
